packages/peregrine/lib/talons/OrderHistoryPage/orderRow.gql.js

```diff
@@ -1,8 +1,8 @@
 import { gql } from '@apollo/client';
 
-export const GET_PRODUCT_THUMBNAILS_BY_SKU = gql`
-    query GetProductThumbnailsBySku($skus: [String!]!) {
-        products(filter: { sku: { in: $skus } }) {
+export const GET_PRODUCT_THUMBNAILS_BY_URL_KEY = gql`
```

Comment:
Had to change this over to query by URL Key, since the product_sku returned by orders was variant skus which will return no results from the backend. Issue was escalated to GraphQL team, and they will be adding new fields in 2.4.2 so we shouldn't need to make a secondary query to get this data.

Reply:
> which will return no results from the backend.

Is this because of our sample data or is it by design of the API?

Reply:
That is by design; configurable variants all have Visibility = Not Visible Individually, so GraphQL is correct in not returning them. Couple JIRA tickets were created to address this though; [this one](https://jira.corp.magento.com/browse/MC-38534) so we don't even need to do a second query for images, and [this one](https://jira.corp.magento.com/browse/MC-38536) so parent sku is returned (which is also a blocker to doing re-orders).